flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/plan/schema/DataStreamTable.scala

```diff
@@ -18,22 +18,11 @@
 
 package org.apache.flink.api.table.plan.schema
 
-import org.apache.calcite.rel.`type`.{RelDataType, RelDataTypeFactory}
-import org.apache.flink.api.table.FlinkTypeFactory
 import org.apache.flink.streaming.api.datastream.DataStream
 
 class DataStreamTable[T](
     val dataStream: DataStream[T],
     override val fieldIndexes: Array[Int],
     override val fieldNames: Array[String])
   extends FlinkTable[T](dataStream.getType, fieldIndexes, fieldNames) {
-
-  override def getRowType(typeFactory: RelDataTypeFactory): RelDataType = {
```

Comment:
Because this override method is almost the same with super method.  I remove it to keep consistent with `DataSetTable` which do not override the `getRowType` method.